apps/prairielearn/src/tests/courseEditor.test.ts

```diff
@@ -296,14 +297,32 @@ const testEditData = [
   },
 ];
 
+const publicCopyTestData = [
+  {
+    url: `${baseUrl}/public/course/2/question/2/preview`,
+    formSelector: 'form[name="copy-question-form"]',
```

Comment:
I assume this is why you added `formName` support to the modal? You could also do `form.js-copy-question-form` to avoid changes there.